doc/source/workflow.rst

```diff
@@ -0,0 +1,726 @@
+   this directory for testing purposes. The file system contains an additional
+   directory named :file:`/image` that is not present in a regular system. It
+   contains information KIWI requires during the create step, including a copy
+   of the :file:`config.xml` file.
+
+   Do not make any changes to the system, since they will get lost when
+   re-running the ``prepare`` step again. Whats more, you may introduce errors
+   that will occur during the ``create`` step, that are difficult to track. The
+   recommended way to apply changes to the unpacked image directory is to change
+   the configuration and re-run the ``prepare`` step.
+
+
+The Create Step
+...............
+
+The successful completion of the ``prepare`` step is a prerequisite for the
+``create`` step. It ensures the unpacked root tree is complete and consistent.
+Creating the packed, or final, image is done in the ``create`` step. Multiple
+images can be created using the same unpacked root tree. It is, for example,
+possible to create a self installing OEM image and a virtual machine image from
+a single unpacked root tree. The only prerequisite is that both image types are
+specified in the :file:`config.xml` before the prepare step is executed.
+
+During the ``create`` step the following major operations are performed by
+kiwi:
+
+#. **Execute the User-defined Script** ``images.sh``.
+
+   At the beginning of the image creation process the script named
+   :file:`images.sh` is executed if present. It is executed on the top level of
+   the target root tree. The script is usually used to remove files that are no
+   needed in the final image. For example, if an appliance is being built for a
+   specific hardware, unnecessary kernel drivers can be removed using this
+   script. 
+   
+#. **Create Requested Image Type.** 
+
+   The image types that can be created from a prepared image tree depend on the
+   types specified in the image description file:`config.xml` file. The
+   configuration file must contain at least one ``type`` element. 
+  
+   The currently supported image types are:
+   
+   #. **Live Image**: For CDs,DVDs or flash disks.
+
+   #. **Disk image**: Virtual system disk that can be used in virtual
+      environments such as VMware, Xen, Amazon Cloud, KVM, and others. Depending
+      on the format a guest configuration file is created. 
+
+   #. **OEM Image**: Preload system for install media CD/DVD or flash disk.
+
+   #. **PXE Image**: Network boot image.
+
+   #. **Container Image**: Base images for the Docker engine.
+
+Customizing the Boot Process
+----------------------------
+
+Most Linux systems use a special boot image to control the system boot process
+after the system firmware, BIOS or UEFI, hands control of the hardware to the
+operating system. This boot image is called the initrd. The Linux kernel loads
+the *initrd*, a compressed cpio initial RAM disk, into the RAM and executes
+*init* or, if present, *linuxrc*.
```

Comment:
As these are commands, I'd suggest to use the `:command:` macro for init, and linuxrc and `:file:` for initrd.